### PR TITLE
boxel: Add ControlPanel

### PR DESCRIPTION
**Pull request by backspace**
This is a component meant to support the functionality seen here:

Network Default to ETH then Create Safe - MVP - Zeplin 2022-09-23 12-48-58

Since the indentation there is inconsistent, I chose to align the body with the icon:

Boxel Components 2022-09-23 12-50-02

This is easily changed if it should be otherwise.

You can see it [here](https://boxel.stack.cards/boxel/control-panel-cs-4565/#/docs?s=Components&ss=%3CBoxel%3A%3AControlPanel%3E).

### CODE REVIEW

File: packages/boxel/addon/components/boxel/control-panel/index.gts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    <div class="boxel-control-panel__item-body">
+      {{yield}}
+    </div>
+  </section>
+</template>;
+
+export default class ControlPanel extends Component<Signature> {
+  get jortle(): number {
+    return 3;
+  }
```
Comment (Aierie) [inline packages/boxel/addon/components/boxel/control-panel/index.gts:58]:
```suggestion
```

File: packages/boxel/addon/components/boxel/control-panel/index.gts [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+  <template>
+    <nav class="boxel-control-panel">
```
Comment (Aierie) [inline packages/boxel/addon/components/boxel/control-panel/index.gts:61]: I don't think we can guarantee that this is used as a `nav`. There are various screens that look like they offer actions, not navigation.

File: packages/boxel/addon/components/boxel/control-panel/index.gts (hunk 1 above)
Comment (backspace) [reply packages/boxel/addon/components/boxel/control-panel/index.gts:58]: lol oops thanks

File: packages/boxel/addon/components/boxel/control-panel/index.gts (hunk 2 above)
Comment (backspace) [reply packages/boxel/addon/components/boxel/control-panel/index.gts]: ya, changed now, thanks

**Pull request merged**